BuildResidentialHPXML/measure.rb

```diff
@@ -1384,10 +1384,9 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     # geothermal_loop_configuration_choices << HPXML::GeothermalLoopLoopConfigurationOther
     geothermal_loop_configuration_choices << HPXML::GeothermalLoopLoopConfigurationVertical
 
-    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('geothermal_loop_configuration', geothermal_loop_configuration_choices, true)
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('geothermal_loop_configuration', geothermal_loop_configuration_choices, false)
```

Comment:
Should we just remove the "none" choice altogether? Leaving the choices array with only "vertical" as an option?

Reply:
Either way seems fine. But I just noticed that we can create a `GeothermalLoop` in the HPXML file even if the user didn't select a GSHP for their HVAC system. Shouldn't we check that?